uint256.go

```diff
@@ -234,7 +226,11 @@ func (z *Int) SubOverflow(x, y *Int) bool {
 
 // Sub sets z to the difference x-y
 func (z *Int) Sub(x, y *Int) *Int {
-	z.SubOverflow(x, y) // Inlined.
+	var carry uint64
+	z[0], carry = bits.Sub64(x[0], y[0], carry)
```

Comment:
For consistency with `Add()` use `0` explicitly.
```suggestion
	z[0], carry = bits.Sub64(x[0], y[0], 0)
```